rotkehlchen/data/all_assets.json

```diff
@@ -8246,14 +8266,24 @@
     },
     "NDX": {
         "coingecko": "newdex-token",
-        "cryptocompare": "",
+        "cryptocompare": "",    
```

Comment:
Ah man you added whitespace here. Can you please remove it? Sucks cause it's all green already.